### ISSUE TITLE
Delete 8.0 branch as it is not supported

### ISSUE THREAD

**Issue by fulldecent** [PrestaShop/PrestaShop#36090]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

The security policy https://github.com/PrestaShop/PrestaShop/security states that only the latest version (currently 8.1) is supported.

Therefore the older, not supported and not in-development branches can be deleted. Currently this includes:

* 8.0.x (no longer supported)
* release-815 (should be a tag, not a branch)
* 1.7.8.x (no longer supported)

as listed at https://github.com/PrestaShop/PrestaShop/branches

### Steps to reproduce

See https://github.com/PrestaShop/PrestaShop/security and https://github.com/PrestaShop/PrestaShop/branches

### PrestaShop version(s) where the bug happened

develop

**Comment by matks** [comment]
Hello @fulldecent why do you want us to delete branches following our own policy?

I don't understand why this topic matters to you. What does it bring to you? I don't see how it affects you what branch we keep open and what branch we don't 😄

**Comment by fulldecent** [comment]
I didn't see the official policy explained at https://github.com/PrestaShop/PrestaShop/blob/develop/CONTRIBUTING.md so I made the assumption above of what the policy should be.

If the policy is otherwise, please let me know and I'll be happy to document it there to give clarity in the future.

---

The specific harm that I think this caused, and why I care is this: https://github.com/PrestaShop/PrestaShop/issues/28989#issuecomment-2093638092

tl;dr Some code about fixing documentation for 1.7 -> 8.x updates was committed to `develop` branch last year but never made its way into the 8.1.5 release.

I think maybe this was caused by some confusion about which code eventually gets into which branches. Spelling out the branch maintenance policy (which I think it is reasonable to match the supported branches in [our Security Policy](https://github.com/PrestaShop/PrestaShop/security)) can help to avoid this and other problems in future.

**Comment by matks** [comment]
1.7.8.x branch is still supported -> https://build.prestashop-project.org/news/2023/178-in-extended-support-phase/

8.0.x is not supported anymore but kept around just in case. We created one extra patch version for 8.0.x in the past even if not supported when we found a critical security issue in it. For emergency cases we go past the support policy 😉 .

release-8.1.5 can be deleted without worries 👍

**Comment by nicosomb** [comment]
I deleted `release-815` branch.

**Comment by jolelievre** [comment]
Hi @fulldecent you're right this branch is not really supported anymore but as @matks we still keep it a few more months just in case a critical security fix is needed (it happened once so far). I think we'll wait for the v9.0 to be released and then we'll clean the branches

**Comment by jolelievre** [comment]
By the way the 1.7.8.x branch is still needed though because it's in LTS (long-time support) at least until the v9 release, I approved you modification on the contributing explanation, so I think we can close this issue 😉
